<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgWindowOptionsWidget</class>
 <widget class="QWidget" name="MsgWindowOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Msg Window</string>
  </property>
  <layout class="QHBoxLayout" name="loMsgWindow">
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>18</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>18</number>
   </property>
   <item>
    <widget class="QGroupBox" name="boxMsgWindowSettings">
     <property name="title">
      <string> Message Window settings  </string>
     </property>
     <layout class="QVBoxLayout" name="loMsgWindowSettings">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>18</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <item>
       <widget class="QGroupBox" name="boxMWPosition">
        <property name="title">
         <string> Position  </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QRadioButton" name="rbMWPredefinedPosition">
             <property name="text">
              <string>predefined position</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lblMsgWinX">
             <property name="text">
              <string>X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinMsgWinX">
             <property name="maximum">
              <number>2000</number>
             </property>
             <property name="singleStep">
              <number>20</number>
             </property>
             <property name="value">
              <number>200</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblMsgWinY">
             <property name="text">
              <string>Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinMsgWinY">
             <property name="maximum">
              <number>2000</number>
             </property>
             <property name="singleStep">
              <number>20</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="rbMWShowInLastPosition">
           <property name="text">
            <string>show in the last remembered position</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>9</number>
        </property>
        <item>
         <widget class="QGroupBox" name="boxMWTimeLive">
          <property name="title">
           <string> Time live  </string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QRadioButton" name="rbMWCloseAfter">
               <property name="text">
                <string>close after</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinMWCloseAfter">
               <property name="maximum">
                <number>600</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblMWCloseAfter">
               <property name="text">
                <string>sec</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="rbMWCloseManually">
             <property name="text">
              <string>close manuallly</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="boxMWStyle">
          <property name="title">
           <string> Style </string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item alignment="Qt::AlignHCenter">
            <widget class="QCheckBox" name="chkMWStayOnTop">
             <property name="text">
              <string>Stay on top</string>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
